schema.sql

```diff
@@ -1,5 +1,10 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
+    id integer,
```

Comment:
- [ ] (OPTIONAL) Good job so far 👍 It would be great if you could make the `id` column not null and autoincremented so that you won't have to input the values manually and also mark the `id` column as primary key 💯